avaframe/com4FlowPy/flowCore.py

```diff
@@ -319,6 +319,7 @@ def calculation(args):
     # if not, wait for 30 secs and check again
     # should prevent the occurence of broken pipe errors or similar issues related
     # to RAM overflow
+    log = logging.getLogger(__name__)
```

Comment:
log is never used, remove it? Or is this for a planned usage?

Reply:
used it for debugging and forgot to remove it